.env

```diff
@@ -0,0 +1 @@
+GLOBAL_ENVIRONMENT="INFO"
```

Comment:
Look onto my comment about the namings please. In this context, instead of info you should write `DEV`